### PR TITLE
Support other dbs

### PR DESCRIPTION
**Pull request by stepmr**
django-twitter-stream only supports mysql.  I’ve added basic support for Postgres.  For other databases, StatusView fallsback to using “created_at” for “time”.

### CODE REVIEW

File: twitter_stream/models.py [hunk 1]
```diff
@@ -299,15 +300,18 @@ def count_approx(cls):
...
-        row = cursor.fetchone()
-        row = dict(zip([col[0].lower() for col in desc], row))
+            desc = cursor.description
+            row = cursor.fetchone()
+            row = dict(zip([col[0].lower() for col in desc], row))
 
-        return int(row['rows'])
+            return int(row['rows'])
+        else:
+            return len(cls.objects.all())
```
Comment (michaelbrooks) [inline twitter_stream/models.py]: Shouldn't this be `cls.objects.count()` instead? e.g. [this](https://docs.djangoproject.com/en/1.8/ref/models/querysets/#django.db.models.query.QuerySet.count)

Comment (KoreyPeters) [inline twitter_stream/models.py]: It should be that. I'd fix this but I can't. If you accept I'll pull and fix.

Comment (michaelbrooks) [inline twitter_stream/models.py]: I merged it to the `postgresql` branch. You can pull that, fix this, and submit another PR. Thank you!
